pkg/restapi/operation/operations.go

```diff
@@ -359,6 +374,13 @@ func handleAuthError(w http.ResponseWriter, status int, msg string) {
 	}
 }
 
+func (c *Operation) setUserProfile(handle string, r *user.Profile) {
+	c.presDefsLock.Lock()
+	defer c.presDefsLock.Unlock()
+
+	c.userProfileRequest[handle] = r
```

Comment:
Shouldn't we set this in database instead of local map ?

Reply:
@llorllale @aholovko what you guys recommend ?

Reply:
@talwinder50 should use transient store

Reply:
done